app/Http/Controllers/FileentryController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Fileentry;
+use Exception;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Storage;
+
+class FileentryController extends Controller
+{
+    public function __constructor()
+    {
+        //
+    }
+
+    public function save($file)
+    {
+        $extension = $file->getClientOriginalExtension();
+        $filename = str_random(20) . $file->getFilename() . '.' . $extension;
+
+        try {
+            Storage::disk('local')->put($filename, File::get($file));
+        } catch (Exception $e) {
+            return null;
```

Comment:
Instead of returning null. Handle the exception or just return the exception message.